### ISSUE TITLE
Cannot use hardware keyboard to tab through fields in record

### ISSUE THREAD

**Issue by robertsteilberg** [mVax-app/mVax#91]
When creating or editing a record, the hardware keyboard tab button doesn't properly segue through each text field
